ecommerce/auth/schemas.py

```diff
@@ -1,12 +1,17 @@
-from pydantic import BaseModel, Field
+from pydantic import BaseModel, Field, EmailStr
 
 
 class UserOut(BaseModel):
-    id: str
+    id: int
     email: str
 
 
 class UserAuth(BaseModel):

```

Comment:
Rename to `UserRegisterInput`? Making clear distinction betwen input and output types proves to perform well over time as project grows.